src/EventService.Data/CategoryRepository.cs

```diff
@@ -3,16 +3,42 @@
 using LT.DigitalOffice.EventService.Models.Db;
 using Microsoft.EntityFrameworkCore;
 using System.Collections.Generic;
+using System.Threading;
 using System.Threading.Tasks;
 using LT.DigitalOffice.EventService.Data.Interfaces;
 using LT.DigitalOffice.EventService.Data.Provider;
+using LT.DigitalOffice.EventService.Models.Dto.Requests.Category;
 
 namespace LT.DigitalOffice.EventService.Data;
 
 public class CategoryRepository : ICategoryRepository
 {
   private readonly IDataProvider _provider;
 
+  private async Task<IQueryable<DbCategory>> CreateFindPredicates(
+  FindCategoriesFilter filter,
+  Guid categoryId)
```

Comment:
there should be no categoryId in find command. Id required for Get. For now you can find only 0 or 1 categories.